yarn-project/aztec.js/src/contract/deploy_proven_tx.ts

```diff
@@ -25,7 +25,7 @@ export class DeployProvenTx<TContract extends Contract = Contract> extends Prove
    */
   public override send(): DeploySentTx<TContract> {
     const promise = (() => {
-      return this.wallet.sendTx(this);
+      return this.wallet.sendTx(this.getPlainDataTx());
```

Comment:
just checking that this was meant to be in this PR?

Reply:
Yes, this is needed to fix serialization. Signed off by grego